src/wallet/coinselection.cpp

```diff
@@ -172,9 +173,20 @@ util::Result<SelectionResult> SelectCoinsBnB(std::vector<OutputGroup>& utxo_pool
     return result;
 }
 
-util::Result<SelectionResult> SelectCoinsSRD(const std::vector<OutputGroup>& utxo_pool, CAmount target_value, FastRandomContext& rng)
+class MinOutputGroupComparator
+{
+public:
+    int operator() (const OutputGroup& group1, const OutputGroup& group2)
```

Comment:
fd8cf58d736ed9614f46b4f5b3c92f71ff9c46d4
nit
```suggestion
    const int operator() (const OutputGroup& group1, const OutputGroup& group2)
```

Reply:
hmm why?
`warning: 'const' type qualifier on return type has no effect`

Reply:
Whoops I meant `int operator() (const OutputGroup& group1, const OutputGroup& group2) const `